### PR TITLE
Bump legacy ARC chart's app version to v0.27.6

### PR DESCRIPTION
**Pull request by mumoshu**
Bumps the appVersion to 0.27.6.
Please see https://github.com/actions/actions-runner-controller/releases/tag/v0.27.6 for more information about ARC v0.27.6!

### CODE REVIEW

**Review by nikola-jokic — DISMISSED** [review]

**Comment by sarahkadar** [comment]
@mumoshu @nikola-jokic  Hi, is there any update on this? I was gonna install the new version and then I saw the helm chart has not been updated and we use it via Helm.

File: charts/actions-runner-controller/Chart.yaml [hunk 1]
```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.23.5
+version: 0.23.6
 
 # Used as the default manager tag value when no tag property is provided in the values.yaml
-appVersion: 0.27.5
+appVersion: 0.27.6
```
Comment (mumoshu) [inline charts/actions-runner-controller/Chart.yaml:21]:
```suggestion
appVersion: 0.27.7
```

File: charts/actions-runner-controller/Chart.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ type: application
 version: 0.23.6
```
Comment (mumoshu) [inline charts/actions-runner-controller/Chart.yaml]:
```suggestion
version: 0.23.7
```

File: charts/actions-runner-controller/Chart.yaml [hunk 3]
```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.23.6
+version: 0.23.7
 
 # Used as the default manager tag value when no tag property is provided in the values.yaml
-appVersion: 0.27.5
+appVersion: 0.27.7
```
Comment (mumoshu) [inline charts/actions-runner-controller/Chart.yaml]:
```suggestion
appVersion: 0.27.6
```

**Comment by mumoshu** [comment]
I had some confusion while resolving the conflict with #3105! It should be good to go now...

**Review by nikola-jokic — APPROVED** [review]

**Comment by sarahkadar** [comment]
Thanks!

**Pull request merged**